src/components/Layout.tsx

```diff
@@ -1,18 +1,39 @@
 
-import Nav from './Nav'
+import Header from './Header'
+import Footer from './Footer'
+import { Box } from '@mui/material'
+import { useRouter } from 'next/router'
 
 interface Props {
     children: any
 }
 
 const Layout = ({children}: Props) => {
-    return (
-        <>
-            <Nav/>
-            <div>
-                {children}
-            </div>
-        </>
-    )
+    const router = useRouter();
+    if (router.route === '/signin' || router.route === '/signup') {
+        return (
+            <>
+                <Box>
+                    {children}
+                </Box>
+            </>
```

Comment:
```suggestion
                <Box>
                    {children}
                </Box>
```
부모 컴포넌트 하나만 리턴하는거라 빈 태그는 없어져도 될 것 같아요

Reply:
아하! 넵 수정하겠습니😊